microprofile/WFLY-10712_microprofile_metrics.asciidoc

```diff
@@ -0,0 +1,230 @@
+= [WFLY-10712] Support Eclipse MicroProfile Metrics
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          observability,microprofile,metrics,openshift
+
+== Overview
+
+The goal of this feature is to increase observability of WildFly by exposing its metrics to monitoring tools such as https://prometheus.io[Promotheus]
+and the https://docs.openshift.com/container-platform/3.11/architecture/infrastructure_components/web_console.html[OpenShift Console].
```

Comment:
Is the console really consumer of these metrics?

What about https://github.com/kiali/kiali ?

Reply:
I listed both OpenShift Console and Kiali in the interested projects but I am still figuring out which does what and which will actually consume metrics exposed by WildFly

Reply:
As work on this proceeds the discussion of what console to support needs to shift to focus on explicit technology requirements, with particular consumers just being examples of consumers who consume whatever way is spelled out. So how the WF server will be discovered, what protocol, what authentication etc. That's already touched on here to a large degree.